pages/index.js

```diff
@@ -169,11 +200,46 @@ const ChildButton = styled.button`
   border-radius: 5px;
   color: #e9f2ef;
   border: none;
-  width: 50%;
+  width: 7rem;
 
   :hover {
     background: #224024;
     box-shadow: 4px 4px 8px 1px rgba(34, 64, 36, 0.65);
     transform: scale(1.1);
   }
 `;
+
+const LogoutButton = styled.button`
+  background-color: #a61f2b;
+  border-radius: 5px;
+  color: #e9f2ef;
+  border: none;
+  width: 7rem;
+  box-shadow: 4px 4px 8px 1px rgba(166, 31, 43, 0.65);
+
+  :hover {
+    background-color: #a61f43;
+    box-shadow: 4px 4px 8px 1px rgba(166, 31, 67, 0.65);
+    transform: scale(1.1);
+  }
+`;
+
+const ChildLogoutButton = styled.button`
+  background-color: #a61f2b;
+  border-radius: 5px;
+  color: #e9f2ef;
+  border: none;
+  width: 7rem;
+  box-shadow: 4px 4px 8px 1px rgba(166, 31, 43, 0.65);
+
+  :hover {
+    background-color: #a61f43;
+    box-shadow: 4px 4px 8px 1px rgba(166, 31, 67, 0.65);
+    transform: scale(1.1);
+  }
+`;
+
```

Comment:
You have a styled component for every button and they share a lot of code. Please consider to use a single `StyledButton` component and configure it with props. (DRY=don't repeat yourself!)